lib/perl/Genome/Test/Command/RunPteroIntegrationTests.pm

```diff
@@ -2,14 +2,18 @@ package Genome::Test::Command::RunPteroIntegrationTests;
 
 use strict;
 use warnings FATAL => 'all';
-use Genome;
+use above "Genome";
```

Comment:
It doesn't strictly require it, but it is designed to run tests (it is just separate from the .t files so it isn't run as a unit-test).  If you want to run these tests against a checkout it is handy to not have to manipulate your PERL5LIB.